protocols/direct-fund/direct-fund.go

```diff
@@ -243,17 +242,6 @@ func (s DirectFundObjective) safeToDeposit() bool {
 	return true
 }
 
-// amountToDeposit computes the appropriate amount to deposit given the current recorded OnChainHoldings
-func (s DirectFundObjective) amountToDeposit() types.Funds {
```

Comment:
My understanding is that `s.C.OnChainFunding` returns the funding per `asset` for a channel. This means that `holdings` could be due to our deposit or another particpant.

I think this means `big.NewInt(0).Sub(s.myDepositTarget[asset], holding)` won't be correct if other participants have already deposited since `holding` will reflect their deposit.

Reply:
This was incorrect. We were just not iterating over deposit targets.